sklearn/datasets/_openml.py

```diff
@@ -768,6 +771,10 @@ def fetch_openml(name=None, *, version='active', data_id=None, data_home=None,
     if data_description['format'].lower() == 'sparse_arff':
         return_sparse = True
 
+    if as_frame == 'auto':
+        # GH#14888: 'auto' means returning frame unless data is sparse
```

Comment:
Once this is the defined API, as described in the docstring, it does not need reference to the issue, IMO.

Reply:
makes sense, thanks for review!